playbook/appherd/roles/create_env_settings/templates/aws-env.py.j2

```diff
@@ -226,25 +192,10 @@ USER_ID_ITERATIONS = {{ django_user_id_iterations }}
 SETTINGS_AUTH_USER = 'ben'
 SETTINGS_AUTH_PASSWORD = 'pbkdf2_sha256$24000$V6XjGqYYNGY7$13tFC13aaTohxBgP2W3glTBz6PSbQN4l6HmUtxQrUys='
 
-ORGANIZATION_NAME = "{{ django_org_name }}"
```

Comment:
What did this do before? I'm just curious.

Reply:
This is just being removed because it's a duplicate overwriting base.py. It's the name of the application, used in a couple of display strings